package.json

```diff
@@ -28,6 +28,7 @@
   },
   "homepage": "https://github.com/googlechrome/big-rig#readme",
   "dependencies": {
+    "axe-core": "^1.1.1",
     "chrome-devtools-frontend": "1.0.381789",
```

Comment:
Is there a reason we're opting for the axe-core audits vs reusing https://github.com/GoogleChrome/accessibility-developer-tools for a11y audits?